tests/runner-tests/expr/undecl/0.c

```diff
@@ -0,0 +1,11 @@
+// errors: 7
```

Comment:
Since 'undeclared' is only used in one or two places in the compiler, can you change the directory name to use the full word instead of abbreviating it? `expr` is abbreviated because it's used so much but I had to think about `undecl` for a second.

Reply:
a0ba82511af385495f766899764dac8c7913db97